src/CommandBasedBackendRegisterAccessor.cpp

```diff
@@ -24,28 +24,28 @@ namespace ChimeraTK {
     _isRecoveryTestAccessor(isRecoveryTestAccessor), _backend(boost::dynamic_pointer_cast<CommandBasedBackend>(dev)) {
     assert(_registerInfo.getNumberOfChannels() != 0);
     assert(_registerInfo.getNumberOfElements() != 0);
-    assert(_registerInfo.getNumberOfDimensions() < 2); // implementation only for scalar and 1D
+    assert(_registerInfo.getNumberOfDimensions() < 2); // Implementation is only for scalar and 1D.
 
-    // 0 means all elements descrived in registerInfo
+    // Use 0 to means all elements are described in registerInfo.
     if(_numberOfElements == 0) {
       _numberOfElements = _registerInfo.getNumberOfElements();
     }
     if(_elementOffsetInRegister + _numberOfElements > _registerInfo.getNumberOfElements()) {
       throw ChimeraTK::logic_error("Requested offset + nElemements exceeds register size in " + this->getName());
     }
 
-    flags.checkForUnknownFlags({}); // require no flags.
+    flags.checkForUnknownFlags({}); // Required to have no flags.
```

Comment:
"Required" is wrong here. This code enforces that the user does not put in any flags. Is not a necessity to have no flags, which I would understand from "Required to have". I

Reply:
How about "no flags are required"?

Reply:
Maybe "require" is the wrong word here. At least it seems confusing.

"Ensure that the user has not requested any flags, as none are supported."

The checkForUnknownFlags() gets the list of known/allowed flags and compares this to the list if flags requested by the user. If there are any flags in the user list that is not the list of known flags, it will throw. Here the list of known flags is empty.